### PR TITLE
Use API information to get owner

### LINKED ISSUE #3115: Incorrect owner in search results for prometheus
The GUI shows prometheus as owned by "Charmers" when it should be showing "prometheus-charmers"

https://api.jujucharms.com/charmstore/v5/prometheus/meta/owner

screen shot 2017-07-28 at 10 55 10 am

### PR DESCRIPTION
**Pull request by makyo**
Fixes #3115

### CODE REVIEW

**Comment by makyo** [comment]
# QA

Run the gui and search for `prometheus`; the results should say `prometheus-charmers` instead of `charmers`.

I freely expect this will spawn a discussion about using data from the API vs. the URL.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7945/

File: jujugui/static/gui/src/app/models/entity-extension.js [hunk 1]
```diff
@@ -34,21 +34,6 @@ YUI.add('entity-extension', function(Y) {
   EntityExtension.prototype = {
 
     /**
-      Parse the owner from the ID.
-      @method ownerFromId
-      @return {String} the charm owner.
-     */
-    ownerFromId: function() {
```
Comment (frankban) [inline jujugui/static/gui/src/app/models/entity-extension.js]: \o/ I like to see these kind of functions deleted!

**Comment by frankban** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**